tests/js/common/shell/shell-pregel-basics-example.js

```diff
@@ -230,6 +235,58 @@ function basicTestSuite() {
       if (i === 0) {
         assertTrue(false, "timeout in pregel execution");
       }
+    },
+
+    test_AQL_pregel_result_is_empty_after_ttl_expires: function () {
```

Comment:
not sure if this test should be in here because it sleeps more than 25s which extends the full jenkins runtime. But it documents the behavior and the config parameter 'ttl', therefore I thought it could be good to have it.